services/explorer/src/App.tsx

```diff
@@ -8,40 +8,47 @@ import BlocksPage from './pages/Blocks/Blocks';
 import NodesPage from './pages/Nodes/Nodes';
 import TransactionDataContainer from './pages/TransactionData/TransactionData';
 import BlockDataContainer from './pages/BlockData/BlockDataContainer';
+import FaucetPage from './pages/Faucet/Faucet';
+import { ExplorerLoaderProvider } from './hooks/useLoader';
+import LoadingWidget from './components/LoadingWidget/LoadingWidget';
 
 declare global {
   interface Window {
     gtag: any;
   }
 }
 
-const Root = () => {
+const Root = (): JSX.Element => {
   return (
     <>
       <Header />
-      <Container maxWidth='xl'>
-        <Box sx={{ px: 4 }}>
-          <Routes>
-            <Route path='/' element={<HomePage />} />
-            <Route path='/blocks' element={<BlocksPage />} />
-            <Route path='/blocks/:txHash' element={<BlockDataContainer />} />
-            <Route path='/dag' element={<HomePage />} />
-            <Route path='/transactions' element={<TransactionsPage />} />
-            <Route
-              path='/transactions/:txHash'
-              element={<TransactionDataContainer />}
-            />
-            <Route path='/nodes' element={<NodesPage />} />
-            <Route path='*' element={<Navigate to='/' replace />} />
-          </Routes>
-          <Footer />
-        </Box>
-      </Container>
+      <ExplorerLoaderProvider>
```

Comment:
I would move this in the index.tsx file, as a wrapper for the <App />. We have all the providers there so it's more consistent to move it there.